client/packages/common/src/authentication/api/operations.graphql

```diff
@@ -148,12 +148,9 @@ query permissions($storeId: String!) {
   }
 }
 
-query storePreferences($storeId: String!) {
```

Comment:
storePreferences query wasn't being used anywhere (in all these  UI areas where we use store prefs, we just pull them off store node from `useAuthContext`) - replaced with this prefs query